contxt/services/ems.py

```diff
@@ -29,6 +37,29 @@ def __init__(self, auth: Auth, env: str = "production", **kwargs) -> None:
     def get_facility(self, id: int) -> Facility:
         return Facility.from_api(self.get(f"facilities/{id}"))
 
+    def get_metric_values(
+        self,
+        asset_id: str,
+        metric_label: Optional[str] = None,
+        params: Optional[dict] = None,
+        page_options: Optional[PageOptions] = None,
+    ) -> Iterable[MetricValue]:
+        assert (
+            params is None
+            or sum([key not in ["effective_end_date", "effective_start_date"] for key in params]) == 0
+        ), f"Unrecognized query parameters: {params}"
```

Comment:
thoughts on letting the api handle this validation? if the server changes (i.e. to allow additional params) then this becomes out-of-sync